### ISSUE TITLE
Convert to SpringBoot?

### ISSUE THREAD

**Issue by cstroe** [caelum/mamute#303]
It seems that VRaptor is no longer maintained and this projects integrates many disparate Java libraries that are hard to get working together.

Would it be valuable to convert this project to use the SpringBoot framework?

**Comment by steimntz** [comment]
Sad, but true. I agree because I think more people can get involved on a more maintained stack. VRaptor is great it has awesome features as injecting in a constructor but Spring is always on the pulse of the industry.

**Comment by icemagno** [comment]
Learn with it! This is the price you'll pay for reinvent the weel. Don't ever create a tool for YOU when there is a good tool in the market that do the same.

**Comment by vicenthy** [comment]
great idea

**Comment by vicenthy** [comment]
let's get?

**Comment by cstroe** [comment]
How do we want to start?  Do we make incremental PRs to master?

**Comment by vicenthy** [comment]
no member is present, better create another repository

**Comment by cstroe** [comment]
Alright, let's work on my fork for now: https://github.com/cstroe/mamute.  We will merge to the master branch on that repo, and hopefully cstroe/mamute@master can be merged back into this repo's master when the work is complete.

**Comment by vicenthy** [comment]
cool, the issues will continue here and the commits will go there and what's the next step?

let's continue with the same architecture and just migrate to spring MVC?

**Comment by vicenthy** [comment]
Great! I understand

**Comment by cstroe** [comment]
Work has begun here: https://github.com/mamutehq/mamute/milestone/1

**Comment by icemagno** [comment]
Is that repository up to date? I see 7 releases and very old files..

**Comment by cstroe** [comment]
That repository is a fork of this one, so it kept the releases.  It is not a rewrite from scratch, but an in-place migration (for example, see the current PR that's converting the database migrations to Liquibase: https://github.com/mamutehq/mamute/pull/6


![screenshot from 2018-07-12 04-37-13](https://user-images.githubusercontent.com/1182093/42625738-217682c4-858e-11e8-9e33-e4287d3ac15c.png)

**Comment by marikth** [comment]
Do you have any knowledge in java, if you are willing to help,
We can start a new forum and do a migration,
I can do all the integrations including elastic search and spring and I
have a friend that are willing to code all the UI using rest.

Any one is up for the challenge?

**Comment by cstroe** [comment]
@marikth Are you proposing a complete rewrite of Mamute? In that case, please see https://github.com/caelum/mamute/issues/299, which created https://github.com/JamesSullivan/manmosu

If you have a specific feature in mind (you mention ElasticSearch and adopting a UI framework/exposing a RESTful API), feel free to document your idea as an issue on https://github.com/mamutehq/mamute and open PRs against that repo.

**Comment by icemagno** [comment]
Yeah. I'm a Java developer. But for now I'm in a [very big military project](https://github.com/icemagno/geoinfra/) and have no time left to another mission.
